app/presenters/taxon_presenter.rb

```diff
@@ -63,4 +63,10 @@ def accordion_content
 
     accordion_items
   end
+
+private
+
+  def ordered_child_taxons
+    taxon.world_related? ? WorldTaxonomySorter.call(taxon.child_taxons) : taxon.child_taxons
```

Comment:
This is fairly minor - but I wonder if pushing the `if` into `WorldTaxonomySorter` would make this work easier to "upgrade" when we come up with a more robust solution. If `WorldTaxonomySorter` is completely responsible for deciding when to sort, it pushes the extra logic into one place which makes changing the way (or even the logic of when) we sort a bit easier for the future.

I might be tempted to actually test in `WorldTaxonomySorter` if the `base_path` starts with `/world` - I know it would duplicate code from the `Taxon` model, but it would allow the Taxon model to keep the `world_related?` private - that way, nobody else would be tempted to rely on that portion of the `Taxon` model interface which might make it better in the future if/when we want to make that more generic?

It's probably fairly academic though, so just something to discuss maybe.